### PR TITLE
Add instructions for thoth bots

### PR DESCRIPTION
**Pull request by 4n4nd**
Addressing https://github.com/aicoe-aiops/ocp4-anomaly-detection/issues/1

### CODE REVIEW

**Comment by 4n4nd** [comment]
@harshad16 Thanks!
How is it that sesheta is available in this repo?

**Comment by durandom** [comment]
> we would update zuul shortly and will keep in my notes to update the information here as well.

@harshad16 @goern if there's a way that you maintain this "quickstart for data scientists" in your repos, I think that would be the better way. We would only link to your docs then. Otherwise documentation will get out of sync

**Comment by harshad16** [comment]
> @harshad16 Thanks!
> How is it that sesheta is available in this repo?
> 
> but it isn't available in [this one](https://github.com/aicoe-aiops/project-template/pull/5)

The configuration seems to be issue here. let us have a look at this.

**Comment by harshad16** [comment]
> > we would update zuul shortly and will keep in my notes to update the information here as well.
> 
> @harshad16 @goern if there's a way that you maintain this "quickstart for data scientists" in your repos, I think that would be the better way. We would only link to your docs then. Otherwise documentation will get out of sync

@durandom , agree that documents can get out of sync, we will add teh quickstart document and other requirement document in [aicoe-ci](https://github.com/AICoE/aicoe-ci), so everything remains in one place. 
Thanks for pointing this out.

File: Thoth-bots.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          - "noop"
+      gate:
+        jobs:
+          - "noop"
+      kebechet-auto-gate:
+        jobs:
+          - "noop"
+  ```
+
+* You can add different types of jobs:
```
[the remark this answers is not in the document]
Comment (harshad16) [reply Thoth-bots.md:58]: @tumido as your pointed it the repositories are to be in zuul-config that correct may bad, I skipped commenting on that.
however we are planning to retire zuul, and assistance of service like pull request checks/test and pull request build/push image would be taken care of by [aicoe-ci](https://github.com/AICoE/aicoe-ci) and merging would be optional. 
I will update the documentation for more clarity in the repository. 
Thanks for pointing it out @tumido

**Pull request merged**